src/main/resources/application.conf

```diff
@@ -334,7 +335,7 @@ mantis {
 
   sync {
     # Whether to enable fast-sync
-    do-fast-sync = true
+    do-fast-sync = false
```

Comment:
Why was this flag changed?

Reply:
as spoken, it was wrong change in config